tools/IoTEdgeDevOps/VstsPipelineSync/DictionaryExtension.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace VstsPipelineSync
+{
+    using System.Collections.Generic;
+
+    public static class DictionaryExtension
+    {
+        public static TValue GetIfExists<TKey, TValue>(this Dictionary<TKey, TValue> dict, TKey key)
```

Comment:
Did you mean to do this?
https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.dictionary-2.trygetvalue?view=netframework-4.8

Reply:
Any obvious benefit? I prefer not to use out param.

Reply:
Why is out not desirable?

Reply:
out keyword is to show intent explicitly this parameter will be updated inside the method.  I try my best to avoid changing parameter value of a method.  That's why I prefer not to use out.  Even you can't find this syntax in Java.  It is just my general practice.